charts/ziti-edge-tunnel/values.yaml

```diff
@@ -37,6 +37,19 @@ serviceAccount:
   # If not set and create is true, a name is generated using the fullname template
   name: ""
 
+# Configures the liveness probe for the tunnel pod
+livenessProbe:
+  exec:
+    command:
+      - /bin/bash
+      - -c
+      - /usr/local/bin/ziti-edge-tunnel tunnel_status | grep -c '"Success":true'
```

Comment:
This is the correct path, but it seems less fragile to rely upon the assumption that BASH's executable search PATH includes the binary than a hard-coded path. For example, if we ever changed the filesystem location then this would break, but it's even less likely it would be moved to a location not in PATH.

Also, what if we parsed out the desired value more explicitly, like this, in line 46? This substitution would still work even if the output changes to eliminate the received preamble and response header.

```bash
test "$(ziti-edge-tunnel tunnel_status|sed -E 's/(^received\sresponse\s<|>$)//g'|jq '.Success')" = true
```

Reply:
https://github.com/openziti/ziti-tunnel-sdk-c/pull/818